src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="form_box">
+        <legend>Personal information</legend>
+
+        <div class="formfield">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
```

Comment:
ALL text fields should have disabled autocomplete
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
            autocomplete="off"
          >
```